src/modules/item/selectors.ts

```diff
@@ -94,35 +96,57 @@ export const getEntityByItemId = createSelector<RootState, Entity[], Record<stri
     }, {} as Record<string, Entity>)
 )
 
+const getItemSyncedStatus = (item: Item, entity: Entity | null) => {
+  let status = SyncStatus.UNSYNCED
+  if (!entity) {
+    status = SyncStatus.LOADING
+  } else if (areSynced(item, entity)) {
+    status = SyncStatus.SYNCED
+  }
+  return status
+}
+
+const getStatusForTPW = (item: Item, itemCuration: ItemCuration | null, entity: Entity): SyncStatus => {
+  let status: SyncStatus
+  if (!itemCuration) {
+    status = SyncStatus.UNPUBLISHED
+  } else if (itemCuration.status === CurationStatus.PENDING) return SyncStatus.UNDER_REVIEW
```

Comment:
the formatting here looks a bit weird without `{}` no? maybe we can add them

Reply:
Absolutely :)